src/docdb/tree/DocDBCollectionTreeItemBase.ts

```diff
@@ -58,7 +59,7 @@ export abstract class DocDBCollectionTreeItemBase extends DocDBTreeItemBase<Retr
             const client = this.getDocumentClient();
             await new Promise((resolve, reject) => {
                 client.deleteCollection(this.link, function (err) {
-                    err ? reject(new Error(err.body)) : resolve();
+                    err ? reject(makeError(err)) : resolve();
```

Comment:
You shouldn't need to call `makeError` everywhere. All errors should be caught and handled in one place. 

Can we remove the `.body` part and just just do `reject(err)` and still handle the error as a part of ErrorData here?: https://github.com/Microsoft/vscode-cosmosdb/blob/master/src/extension.ts#L168